modules/xmpp/JingleSessionPC.js

```diff
@@ -25,6 +25,33 @@ const IQ_TIMEOUT = 10000;
  *
  */
 export default class JingleSessionPC extends JingleSession {
+    /**
+     * Parses 'senders' attribute of the video content.
+     * @param {jQuery} jingleContents
+     * @return {string|null} one of the values of content "senders" attribute
+     * defined by Jingle. If there is no "senders" attribute or if the value is
+     * invalid then <tt>null</tt> will be returned.
+     * @private
+     */
+    static parseVideoSenders(jingleContents) {
+        const videoContents = jingleContents.find('>content[name="video"]');
+
+        if (videoContents.length) {
+            const senders = videoContents[0].getAttribute('senders');
+
+            switch (senders) {
```

Comment:
Isn't it better if you do this with `if` instead of switch? It looks a little bit weird to me with the switch because it is binary decision.